src/components/Sponsors/Sponsors.css

```diff
@@ -0,0 +1,64 @@
+.sponsors-container {
+  display: flex;
+  flex-direction: column;
+  text-align: center;
+  margin: 50px 0px;
+}
+
+.sponsors-title {
+  display: flex;
+  justify-content: center;
+  flex-wrap: wrap;
+  margin-bottom: 20px;
+}
+
+.box {
+  display: flex;
+  justify-content: center;
+
+  left: 35px;
+  top: 4326px;
```

Comment:
Avoid absolute positioning - this can cause some weird layout things in the future.